packages/net/dns/auth.pony

```diff
@@ -0,0 +1,5 @@
+use "net"
+
+primitive DNSAuth
```

Comment:
I think this file should be named `dns_auth.pony` (even though it is already in a `dns` subdir), because the principal type declared in the file is `DNSAuth`.